src/frontend/components/StudentLiveAdvertising/StudentLiveRegistration/RegistrationForm/index.spec.tsx

```diff
@@ -75,8 +90,46 @@ describe('<RegistrationForm />', () => {
     ).not.toBeInTheDocument();
   });
 
+  it('hides the field for lti user with email', () => {
+    mockGetDecodedJwt.mockReturnValue({
+      consumer_site: 'consumer site',
+      context_id: 'context id',
+      locale: 'en',
+      maintenance: false,
+      permissions: {
+        can_access_dashboard: false,
+        can_update: false,
+      },
+      resource_id: 'resource id',
+      roles: [],
```

Comment:
to be consistent, if we use roles, it's better to add "student" one, empty doesn't exist (multiple times declared)